packages/theme/src/__tests__/__snapshots__/theme.test.tsx.snap

```diff
@@ -29,6 +29,8 @@ Object {
     "colorBackground": "#FFFFFF",
     "colorBackgroundBody": "#F7F9FF",
     "colorBackgroundComplete": "linear-gradient(360deg, rgba(82, 244, 174, 0.12) 0%, rgba(131, 247, 197, 0.0575) 57.81%, rgba(255, 255, 255, 0) 100%)",
+    "colorBackgroundDecorative01": "#C8F0EF",
```

Comment:
Could we have more descriptive names instead of `01` and `02`?

Reply:
I didn't choose them 😅  But they come from the Design tokens: 
![image](https://user-images.githubusercontent.com/5320963/201312934-5939e3d6-d7ce-4208-b553-68c794e5ce0e.png)

Reply:
The colors were updated to 
```
colorBackgroundDecorativeWeak: "#FFF1DB",
colorBackgroundDecorativeStrong: "#C8F0EF",
```